### PR TITLE
MD-2893: Modified queries involving limit & their baseline

### PR DESCRIPTION
**Pull request by prasadns14**
Query drill3788.q - Table had multiple rows with same partkey, so also ordered by discount
Query 5level_sanity2.q - Table had 8 rows for each partkey, so reduced limit to 8

@Agirish please review

### CODE REVIEW

File: framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q [hunk 1]
```diff
@@ -1 +1 @@
-select ps_supplycost from ps_5level where dir0=2000 and dir1='q1' and dir2='jan' and dir3=1 and dir4=1.0 or dir4=2.0 order by ps_partkey limit 10;
+select ps_supplycost from ps_5level where dir0=2000 and dir1='q1' and dir2='jan' and dir3=1 and dir4=1.0 or dir4=2.0 order by ps_partkey limit 8;
```
Comment (Agirish) [inline framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q]: I did not understand this change. Can you please explain why limit 10 was changed to limit 8?

**Comment by Agirish** [comment]
+1, pending review comments.

File: framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q (hunk 1 above)
Comment (prasadns14) [reply framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q]: The table has 8 rows per partkey. When sorted by partkey, the first 8 rows have partkey value 1 but the next two could be any of the 2 rows having partkey value 2. 
The ones being returned by GA branch are different from apache master.

Comment (Agirish) [reply framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q]: Can't we order by ps_supplycost as well?

Comment (prasadns14) [reply framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q]: Made the changes as suggested

Comment (Agirish) [reply framework/resources/Functional/metadata_caching/partition_pruning/data/5level_sanity2.q]: Thanks!

**Pull request merged**